framework_tvb/tvb/adapters/simulator/simulator_fragments.py

```diff
@@ -170,7 +170,7 @@ class SimulatorFinalFragment(ABCAdapterForm):
 
     def __init__(self, prefix='', project_id=None):
         super(SimulatorFinalFragment, self).__init__(prefix, project_id)
-        self.simulation_name = ScalarField(Attr(str, doc='Name for the current simulation configuration',
+        self.simulation_name = ScalarField(Attr(str, doc='Name for the current simulation configuration', default="default_simulation",
```

Comment:
I would expect the name to be different at every run.
In versions 1.* of TVB-Distribution you can see how it was behaving. and giving names "simulation-x" where x is incremental, a number